packages/trader/src/AppV2/Components/ServicesErrorSnackbar/services-error-snackbar.tsx

```diff
@@ -51,7 +49,6 @@ const ServicesErrorSnackbar = observer(() => {
             : '48px';
     const action_props = {
         actionText: localize('View'),
-        delay: PROLONGATED_DURATION,
```

Comment:
This delay was needed for specific cases, when we show snackbar with action. The aim was to give a user more time to read and click on action button. Was it broken?

Reply:
It won't dissappear until user closes the snackbar. There's no point to have a delay.

Reply:
Ooo, got it, thank you